gossip/src/main/resources/gossip-schema.xml

```diff
@@ -57,7 +57,7 @@
       <field name="senderGossipHeartbeat" id="11" type="uint64"/>
       <field name="senderId" id="12" type="uint16"/>
       <data name="eventType" id="14" type="varDataEncoding"/>
-      <data name="payload" id="15" type="varDataEncoding"/>
+      <data name="variables" id="15" type="varDataEncoding"/>
```

Comment:
Ok here it is not necessary :sweat_smile: but since we remove this anyway just leave it as it is.